locksmith/src/middlewares/signatureValidationMiddleware.ts

```diff
@@ -96,6 +96,30 @@ namespace SignatureValidationMiddleware {
       }
     }
   }
+
+  export const evaluateSignature = (
```

Comment:
Since this returns a function that evaluates a signature, rather than evaluating the signature itself, what do you think about renaming this to something like `makeSignatureEvaluator` or something similar? LG otherwise

Reply:
sounds good, will do

Reply:
I'll approve pending that change